### PR TITLE
Update list numerals to use monospace and update error example content

### PR DESCRIPTION
**Pull request by rmccar**
This PR updates the base font for the Design System to use monospacing so that numbered lists can be correctly formatted. It also updates the content of the error lists in examples and as a side effect of all this it fixes an issue around monospaced numbers for data tables.

### CODE REVIEW

**Comment by rmccar** [comment]
[Issue #783: List numerals should use a monospaced variant of the font to ensure lists line up...](https://trello.com/c/Ul0wP2Pw/527-issue-783-list-numerals-should-use-a-monospaced-variant-of-the-font-to-ensure-lists-line-up)

**Comment by rmccar** [comment]
[Update error pattern](https://trello.com/c/sZmntAeK/530-update-error-pattern)

**Comment by rmccar** [comment]
[Issue #640: Add monospaced numerals for data tables](https://trello.com/c/xwaDSthy/443-issue-640-add-monospaced-numerals-for-data-tables)

File: src/components/lists/_macro.njk [hunk 1]
```diff
@@ -9,7 +9,7 @@
       {% for item in (params.itemsList if params.itemsList is iterable else params.itemsList.items()) %}
           <li class="list__item {{ item.listclass }}" {% if item.current %}aria-current="true"{% endif %}>
               {% if item.url and item.current != true %}
-                  {% if item.index %}{{ loop.index }}. {% endif %}{% if item.prefix %}<span class="list__prefix" aria-hidden="true">{{ item.prefix }}.</span> {% endif %} <a href="{{ item.url }}" class="list__link {{ item.classes }}" {% if item.index %}data-qa="list-item-{{ loop.index }}"{% endif %} {% if item.target %} target="{{ item.target }}"{% endif %} {% if item.name %} name="{{ item.name }}"{% endif %}>
+                  <span class="list__prefix" aria-hidden="true">{% if item.index %}{{ loop.index }}. {% endif %}{% if item.prefix %}{{ item.prefix }}. {% endif %}</span><a href="{{ item.url }}" class="list__link {{ item.classes }}" {% if item.index %}data-qa="list-item-{{ loop.index }}"{% endif %} {% if item.target %} target="{{ item.target }}"{% endif %} {% if item.name %} name="{{ item.name }}"{% endif %}>
```
Comment (boxadesign) [inline src/components/lists/_macro.njk:12]: We are outputting the prefix before checking for the prefix param...is this right?

Comment (rmccar) [reply src/components/lists/_macro.njk:12]: Yeah thats probably not right. I just wanted to make sure that if it is set using the `{{ loop.index }}` then it is also included in a span and has the correct classes. Ill rework this

File: src/scss/base/_fonts.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@include font-face(OpenSans, '#{$static}/fonts/opensans-regular', $weight: 400, $style: null, $exts: woff2 woff);
-@include font-face(OpenSans, '#{$static}/fonts/opensans-bold', $weight: 600, $style: null, $exts: woff2 woff);
+@include font-face(OpenSans, '#{$static}/fonts/opensans-regular', $weight: 200, $style: null, $exts: ttf);
```
Comment (jrbarnes9) [inline src/scss/base/_fonts.scss:1]: I think we probably want to keep both woff and ttf formats and only apply to monospaced numerals format to ordered lists and table cells?

**Comment by rmccar** [comment]
> Can you update error validation documentation `The error `panel__header` must display the number of errors on the page _if there is more than one_.`?

I'm not sure I like the "if there is more than one" being bold but ill push it and see what you think

**Pull request merged**